<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomDialog</class>
 <widget class="QDialog" name="CustomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>200</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>自定义 Mine Sweeper</string>
  </property>
  <widget class="QPushButton" name="doneButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>152</y>
     <width>80</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="widthLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请输入界面宽度（10~40）</string>
   </property>
  </widget>
  <widget class="QLabel" name="heightLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请输入界面高度（10~25）</string>
   </property>
  </widget>
  <widget class="QLabel" name="mineLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>请输入地雷数量（1~999）</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="widthSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>28</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::UpDownArrows</enum>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="heightSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>58</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>25</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="mineSpinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>88</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
